pkgs/by-name/pr/prism-model-checker/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  prism-model-checker-unwrapped,
+  java,
+  symlinkJoin,
+  makeWrapper,
+  writeText,
+}:
+let
+  desktopEntry = writeText "prism-model-checker-xprism.desktop" ''
+    [Desktop Entry]
+    Type=Application
+    Icon=prism-model-checker
+    Terminal=false
+    Categories=Science;Math
+    Comment=Probabalistic Symbolic Model Checker
+    Name=XPrism
+    Version=${prism-model-checker-unwrapped.version}
+    Exec=xprism
+  '';
+in
+symlinkJoin {
+  name = "prism-model-checker-${prism-model-checker-unwrapped.version}";
+  nativeBuildInputs = [ makeWrapper ];
+  paths = [
+    prism-model-checker-unwrapped
+    java
+  ];
+
+  postBuild = ''
+    rm -rf $out/bin
+    mkdir --parents $out/bin
+    for f in $(ls ${prism-model-checker-unwrapped}/bin); do
```

Comment:
```suggestion
    for f in ${prism-model-checker-unwrapped}/bin/*; do
```

Reply:
this does not do the same thing. Your change would give the whole path, while I only want the filename

Reply:
Yeah, but using ls's output is [still a bad idea](http://mywiki.wooledge.org/ParsingLs). Just use `$(basename $f)` if you just want the filename.